dependency_tree.yml

```diff
@@ -268,7 +268,7 @@ danu:
   master_branch: main
   requirements_path: requirements.txt
   test_cmd: "python -m pytest"
-  conda_deps: numpy scipy pcraster gdal netCDF4 pandas dask lz4 eccodes magics
+  conda_deps: numpy<2 scipy pcraster gdal netCDF4 pandas dask lz4 eccodes magics
```

Comment:
Hi @figi44 , when I added this restriction on numpy I got a failure on the workflow while building the environment. It looks like it's not parsing the condition string properly.

from the job output
```
+ conda install -y python=3.10 numpy scipy pcraster gdal netCDF4 pandas dask lz4 eccodes magics -c conda-forge
/var/spool/slurmd/job34718883/slurm_script: line 53: 2: No such file or directory
```

Reply:
Hi @colonesej , try putting quotes around "numpy<2", I think that might help

Reply:
Thanks @iainrussell, the build process has been working for a while now.  Can this be merged?